### DISCUSSION TITLE
Programming Help

### DISCUSSION THREAD

**Discussion by devradheee** [community/community#70684] [Programming Help] [answered]
### Body

Conversations related to programming languages, open source, and software development. Please review the pinned post for this category’s guidelines prior to posting.

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by Gyanendu01** [comment, marked as answer]
Hey devradheee,
You can ask Your Question
